human_experiments/scripts/rename_image_timestamp

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+__rename_image_timestamp() {
+    local monitoring_path=$1
+
+    echo "Renaming images in experiment $monitoring_path"
+
+    ssh cat@cat "~/Downloads/FileChecker/scripts/timestampRename.py -d $monitoring_path/lion/face_images/"
```

Comment:
All of our scripts should be versioned. I assume `timestampRename.py` is in a repo, so it should be under `~git/<repo_name>` like we have the tomcat scripts under `~git/tomcat`. Then replace `~/Downloads/FileChecker/scripts/timestampRename.py` with `\$HOME/$SYSTEMS_DIR/<repo_name>/FileChecker/scripts/timestampRename.py`. The variable `$SYSTEMS_DIR` contains the path to our programs, which is currently `git` and `\$HOME` will expand home in the target machine (cat in this case) instead of the machine we are invoking ssh from (tom).

Reply:
I agree with your suggestions and @Vatsav14 and I already planned on doing this at some point. However, I would like this pull request to be incorporated without placing the `FileChecker` program in the repository because I would like to avoid having to come to the lab and manually starting and stopping the image timestamp processes. It is an important part that helps us avoid losing image data, and it is a little complicated to set up.

Reply:
It's fine to merge it now. Let us keep in mind to move it to the repo soon; otherwise, we will be caught up with other stuff.